focus-android/docs/Home.md

```diff
@@ -29,13 +29,13 @@ https://www.mozilla.org/firefox/focus/
 
 ## Developer Documentation
 
-* [Architecture decisions](https://github.com/mozilla-mobile/focus-android/wiki/Architecture-Decisions)
-* [Automation](https://github.com/mozilla-mobile/shared-docs/blob/master/android/automation.md)
-* [Content blocking](https://github.com/mozilla-mobile/focus-android/wiki/Content-blocking)
-* [Multisession architecture](https://github.com/mozilla-mobile/focus-android/wiki/Multisession-architecture)
+* [Architecture decisions](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Architecture-Decisions.md)
+* [Automation](https://github.com/mozilla-mobile/firefox-android/blob/main/docs/shared/android/automation.md)
+* [Content blocking](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Content-blocking.md)
+* [Multisession architecture](https://github.com/mozilla-mobile/firefox-android/blob/main/focus-android/docs/Multisession-architecture.md)
 * [Localization](https://github.com/mozilla-mobile/focus-android/wiki/Localization)
-* [Telemetry](https://github.com/mozilla-mobile/focus-android/wiki/Telemetry)
+* [Telemetry](https://github.com/mozilla-mobile/firefox-android/blob/d8939c35d509cb4839030a318db97025674aea20/focus-android/docs/Telemetry.md)
 
 ## Product Management
-* [Roadmap](https://github.com/mozilla-mobile/focus-android/projects/2)
+* [Roadmap](https://github.com/mozilla-mobile/firefox-android/projects?query=is%3Aopen)
```

Comment:
We can just remove this bullet entirely since we are not doing explicit project tracking in Github or Bugzilla.